### ISSUE TITLE
Customizer: Kategorien/Schlagwörter ausschließen

### ISSUE THREAD

**Issue by diesocke** [Piratenpartei/Pirate-Rogue#184]
Im Customizer können Kategorien und Schlagwörter für bestimmte Bereiche zugewiesen werden. Leider gibt es derzeit keine Möglichkeit, bestimmte dieser Merkmale mind. Die Anzeige aktueller Beiträge auszuschließen. Dadurch werden diese Inhalte ggf. doppelt angezeigt.

**Comment by xwolfde** [comment]
Ja, dazu fehlt mir auch ein Lösungsansatz, der genügend usable in der Bedienung ist. :/

**Comment by diesocke** [comment]
Evtl. per Haken anbieten, in bestimmten anderen Bereichen gewählte Kategorien/Tags abzuwählen?

**Comment by diesocke** [comment]
Evtl. per Haken anbieten, gewählte Kategorien/Tags abzuwählen? Gab es bei den Piratenkleidern ähnlich. Evtl. ließe sich das sogar auf die in anderen Bereichen wie Featured oben/unten benutzten Inhalte begrenzen (so dass nur der Bereich gewählt werden muss, dessen Inhalte nicht mitgeteilt werden, statt einzelner Kategorien/Tags).
